tests/test_collector.py

```diff
@@ -59,15 +59,45 @@ def test_extract_metrics():
 @responses.activate
 def test_get_azure_data():
 
-    enrollment='12345'
+    enrollment = '123'
+    base_url = "https://ea.azure.com/rest/{}/usage-report".format(enrollment)
+    params = "?month={}&type=detail&fmt=Json".format(current_month())
+
     c = AzureEABillingCollector('cloud_costs', enrollment, 'abc123xyz')
 
     responses.add(
         method='GET',
-        url="https://ea.azure.com/rest/{}/usage-report?month=2017-03&type=detail&fmt=Json".format(enrollment),
+        url=base_url+params,
         match_querystring=True,
         json=sample_data
     )
 
-    data = c._get_azure_data('2017-03')
+    data = c._get_azure_data(current_month())
     assert data == sample_data
+
+
+@responses.activate
+def test_empty_month():
+    """
+    If no usage details have are available for a given month the API does not return a JSON document.    
+    """
+    enrollment = '123'
+    base_url = "https://ea.azure.com/rest/{}/usage-report".format(enrollment)
+    params = "?month={}&type=detail&fmt=Json".format(current_month())
+
+    c = AzureEABillingCollector('cloud_costs', enrollment, 'abc123xyz')
+
+    empty_month_data = """"Usage Data Extract",
```

Comment:
adressed in https://github.com/blue-yonder/azure-cost-mon/pull/10/commits/67c42396aad41a856e7bee69dacb12087e03121d